privacy-statement.md

```diff
@@ -0,0 +1,205 @@
+  and of Wikipedia. We require email addresses to be validated in order to process your request; this is accomplished by
+  sending an automated confirmation email with a special link that must be clicked to process your request.
+- Browser characteristics, such as your [User-Agent string][enwiki-user-agent] and [Client Hints][enwiki-client-hints].
+  We collect this information in order to identify and protect against abuse of the Tool and of Wikipedia, and it cannot
+  be accessed by all Account Creators. Only Checkusers, Stewards, and Tool Roots have access to this information.
+
+We take reasonable measures to protect your personal information. Except for the username you request, all the personal
+information above is automatically and permanently deleted from the Tool when it is no longer needed, either:
+
+1. After 15 days from when the request is initially submitted
+2. When the request is fulfilled or declined
+
+...Whichever comes _later_.
+
+We also allow Requesters to submit a free-text comment with their request so that they may add context to their request.
+This comment is visible only to Account Creators and is also permanently linked to your request identifier, as are any
+internal comments left on a request by Account Creators. While these comments are not automatically deleted,
+Administrators are responsible for and endeavor to permanently remove any private or personally identifying information,
+including types of information not explicitly noted above, contained in those requests when the request is closed, and
+we likewise encourage Requesters to avoid including unnecessary personal information in their comments. Account Creators
+are prohibited from adding any Requesters' personal information in their comments, and may be suspended from the Tool if
+they do so.
+
+In order for Account Creators to provide support to Requesters, to enforce Wikipedia policy with respect to usernames
+and account creation, and in order to detect, prevent, and respond to abuse of the Tool and of Wikipedia, Account
+Creators may share some of the information collected with your request with services hosted by the Wikimedia Foundation,
+with other WMCS services, or with external third-party services; for example, your IP address may be used with a
+third-party analysis service to determine if it is a known proxy or VPN IP address; or, if your email address is using
+an uncommon service, we may look up the domain portion (the part after the `@`) in public Internet registries. In these
+cases, we take care to ensure that your unique request identifier and your requested username are not shared with these
+services, so that there is no way for that service to associate the information provided to it with your account request
+or Wikipedia username. This data is also never automatically shared; the action must be initiated by an Account Creator
+and is logged for auditing purposes.
```

Comment:
Careful - yes, it's technically logged via apache request logs, but we don't persist that data for long and it's not always going to be possible to tie that back to an individual user. I think it's probably fine as a v1 for the deadline tonight, but I would like us to reword this in a v2.